src/fides/api/ctl/routes/admin.py

```diff
@@ -21,6 +24,12 @@ async def db_action(action: DBActions) -> Dict:
     """
     Initiate one of the enumerated DBActions.
     """
+
+    if not CONFIG.database.sync_database_uri:
+        raise HTTPException(
+            status_code=HTTP_400_BAD_REQUEST, detail="No database uri provided"
```

Comment:
I don't think this is possible, since we set valid defaults in the config

Reply:
In the current state it has to be checked or mypy will error since `sync_databse_uri` is `Optional`. However, if we make it required as discussed this check can go away.

Reply:
that's what I suspected, thanks for confirming!